raiden-cli/src/routes/connections.ts

```diff
@@ -1,20 +1,133 @@
+ * We don't want to improve this (for now), since SDK doesn't have the concept of connection
+ * manager; it may change when we have proper auto-pilot (#211); before that, this dummy
+ * implementation should enough to make WebUI work with it
+ */
+async function connectTokenNetwork(
+  this: Cli,
+  request: Request,
+  response: Response,
+  next: NextFunction,
+) {
+  if (request.body.initial_channel_target && request.body.initial_channel_target != 1)
+    return response.status(409).send('CLI only supports managing connection with one hub');
+  if (request.body.joinable_funds_target && request.body.joinable_funds_target != 1)
+    return response.status(409).send('CLI only supports allocating whole funds');
+  const token: string = request.params.tokenAddress;
+  try {
+    const channelsDict = await this.raiden.channels$.pipe(first()).toPromise();
+    const hub = await this.raiden.resolveName('hub.raiden.eth');
+    const channel = channelsDict[token]?.[hub];
+    if (!channel) {
+      await this.raiden.openChannel(token, hub, {
+        deposit: request.body.funds,
+      });
+    } else if (
+      request.body.funds &&
+      channel.ownDeposit.sub(channel.ownWithdraw).lt(request.body.funds)
+    ) {
+      // if channel exists but allocated funds are lower than requested, deposit difference
+      // amount = funds - (ownDeposit - ownWithdraw) = ((ownDeposit - ownWithdraw) - funds) * -1
+      const amount = channel.ownDeposit.sub(channel.ownWithdraw).sub(request.body.funds).mul(-1);
+      await this.raiden.depositChannel(token, hub, amount);
+    }
+    response.status(204);
+  } catch (error) {
+    if (isInvalidParameterError(error)) response.status(400).send(error.message);
+    else if (isInsuficientFundsError(error)) response.status(402).send(error.message);
+    else if (isConflictError(error)) response.status(409).send(error.message);
+    else next(error);
+  }
+}
+
+/**
+ * Closes all closeable channels in a token network
+ * TODO: implement auto-settle (see discussion at #237)
+ */
+async function disconnectTokenNetwork(this: Cli, request: Request, response: Response) {
+  const token: string = request.params.tokenAddress;
+  const channelsDict = await this.raiden.channels$.pipe(first()).toPromise();
+  if (!channelsDict[token]) return response.status(404).send('No channels on tokenNetwork');
+  const closeableChannels = Object.values(channelsDict[token]).filter((channel) =>
+    [ChannelState.open, ChannelState.closing].includes(channel.state),
+  );
+  const promises: Promise<string | undefined>[] = [];
+  for (let i = 0; i < closeableChannels.length; i++) {
+    // if more than one channel to close, wait 1s between each call to avoid nonce races
+    if (i > 0) await new Promise((resolve) => setTimeout(resolve, 1000));
+    const channel = closeableChannels[i];
+    promises.push(
+      this.raiden.closeChannel(token, channel.partner).then(
+        () => channel.partner,
+        () => undefined,
```

Comment:
So we simply ignore any closing errors? :thinking:

Reply:
They are logged, and the errored channels don't go to the output. The endpoint doesn't specify http errors for closing errors, so it remains on user's hand to be aware that the channel didn't close and possibly try later.